### DISCUSSION TITLE
Visual Studio 2022 license for react native windows

### DISCUSSION THREAD

**Discussion by V-M-Aravind** [microsoft/react-native-windows#13433] [Q&A] [answered]
Hi everyone. I'm new to React native Windows. I'm building an application for an organization. RNW requires the build tools from visual studio, which is available from community, professional  or enterprise version. We are not using the Visual studio code editor. Its only required for the build purpose right?. In that case do I need to get a license for visual studio paid versions or is it okay to continue with the community version for generating build and distributing the package for production as well.

**Comment by chrisglein** [comment, marked as answer]
The community version should meet your needs to get started. Depending on the size of your organization, I'd check the license agreement. But absolutely you can build RNW apps using just the community version.

**Reply by V-M-Aravind** [reply]
Thank you @chrisglein
